src/modules/CreateDrops/TokenDrop/CreateTokenDropSummary.tsx

```diff
@@ -1,11 +1,19 @@
+import useSWR from 'swr';
 import { DropSummary } from '../DropSummary';
 
 import { useCreateTokenDropContext } from './CreateTokenDropContext';
 
 export const CreateTokenDropSummary = () => {
   const { getSummaryData, getPaymentData, createLinksSWR } = useCreateTokenDropContext();
+
   const summaryData = getSummaryData();
-  const paymentData = getPaymentData();
+  const { data: paymentData, error, isLoading } = useSWR('drops/token/new', getPaymentData);
+  if (error) {
+    console.warn(error)
+    return <div>failed to load</div>
+  }
+  if (isLoading) return <div>loading...</div>
```

Comment:
Just a suggestion, probably better to use a spinner animation for loading? you can use <Spinner /> from Chakra